rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -137,8 +138,8 @@ class AsyncParametersClient
   set_parameters(
     std::vector<rclcpp::parameter::ParameterVariant> parameters,
     std::function<
-      void(std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)
-    > callback = nullptr)
+      void(std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)> callback =
+    nullptr)
```

Comment:
May be we can take a look at your machine / workspace to figure out why it keeps reformatting this code block.